mgmt/rest/metric.go

```diff
@@ -81,17 +81,6 @@ func (s *Server) getMetricsFromTree(w http.ResponseWriter, r *http.Request, para
 		return
 	}
 
-	// If no version was given, get all that fall at this namespace.
```

Comment:
Removing this block of code no longer returns metrics from the rest server when using `curl http://localhost:8181/v1/metrics/intel/mock/foo` and now returns a 400 for invalid syntax instead of versions 1 and 2 of the metric.